app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteRemoveFragment.kt

```diff
@@ -29,16 +30,18 @@ class AutocompleteRemoveFragment : AutocompleteListFragment() {
     }
 
     private fun removeSelectedDomains(context: Context) {
-        launch(CommonPool) {
-            val domains = (domainList.adapter as DomainListAdapter).selection()
-            if (!domains.isEmpty()) {
-                CustomAutocomplete.removeDomains(context, domains)
-
-                TelemetryWrapper.removeAutocompleteDomainsEvent(domains.size)
+        val domains = (domainList.adapter as DomainListAdapter).selection()
+        if (domains.isNotEmpty()) {
+            launch(UI) {
```

Comment:
I think the coroutine code can be simplified. Currently, it's:
```kotlin
            launch(UI) {
                async {
                    CustomAutocomplete.removeDomains(context, domains)

                    TelemetryWrapper.removeAutocompleteDomainsEvent(domains.size)
                }.await()

                fragmentManager.popBackStack()
            }
```

Functionally equivalent, I think this is more readable:
```kotlin
            launch {
                CustomAutocomplete.removeDomains(context, domains)
                TelemetryWrapper.removeAutocompleteDomainsEvent(domains.size)
                launch(UI) {
                    fragmentManager.popBackStack()
                }
            }
```

But it seems `fragmentManager.popBackStack()` could be thread-safe (it queues the popping and returns) so I believe:
```kotlin
            launch {
                CustomAutocomplete.removeDomains(context, domains)
                TelemetryWrapper.removeAutocompleteDomainsEvent(domains.size)
                fragmentManager.popBackStack()
            }
```

Should also work.